MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md

```diff
@@ -0,0 +1,273 @@
+
+You can place radial menu anywhere on its parent layout, and drag it with in the parent layout. 
+
+## Dragging RadialMenu
+
+You can enable/disable dragging by using the [`IsDragEnabled`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_IsDragEnabled) property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:RadialSample"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.RadialMenu;assembly=Syncfusion.Maui.RadialMenu"
+             x:Class="RadialSample.MainPage">
+    <ContentPage.Content>
+        <syncfusion:SfRadialMenu x:Name="radialMenu" 
+                                 IsDragEnabled="true" 
+                                 CenterButtonText="&#xe710;"
+                                 CenterButtonFontFamily="Maui Material Assets" 
+                                 CenterButtonRadius="30"
+                                 CenterButtonFontSize="26"
+                                 CenterButtonBorderThickness="3">
+        </syncfusion:SfRadialMenu>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.RadialMenu;
+
+namespace RadialSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            SfRadialMenu radialMenu = new SfRadialMenu()
+            {
+                IsDragEnabled = true,
+                CenterButtonText="&#xe710;",
+                CenterButtonFontFamily="Maui Material Assets" ,
+                CenterButtonRadius = 30,
+                CenterButtonFontSize = 26,
+                CenterButtonBorderThickness = 3
+            };
+            this.Content = radialMenu;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Image for dragging](images/placing-dragging/maui-radialmenu-dragging.png)
+
+## DragEvents
+
+[`SfRadialMenu`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html) provides for event for [`DragBegin`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_DragBegin) and [`DragEnd`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_DragEnd) in which the event get hooked when the RadialMenu is get dragged.
+
+### DragBegin event
+
+This event get triggered when RadialMenu is start to drag with [`DragBeginEventArgs`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html).
+
+* [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui__SfRadialMenu_DragBeginEventArgs_Position) - Gets the Start position of the RadialMenu
```

Comment:
* [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui__SfRadialMenu_DragBeginEventArgs_Position) - Gets the Start position of the RadialMenu.